README.md

```diff
@@ -48,12 +62,33 @@ import SwiftCurrent
 launchInto(Workflow(ExampleViewController.self))
 ```
 
+### Original:
+
 And just like that you're started!  To see something more practical and in-depth, check out the example app in the repo.  For a more in-depth starting guide, checkout out our [Getting Started](https://github.com/wwt/SwiftCurrent/wiki/getting-started) documentation.
 
+### Option #1:
+
+And just like that you're started!  If you would like to see a more practical example, [checkout the example app in the repo]().  If you would like to have a slightly larger example than this quick start that may answer a number of initial questions you might have, take a look at the getting started guide that is most relevant to you:
+
+- [Getting Started with Storyboards](https://github.com/wwt/SwiftCurrent/wiki/getting-started)
+- [Getting Started with Programmatic UIKit](https://github.com/wwt/SwiftCurrent/wiki/Getting-Started-with-Programmatic-UIKit)
+
+
+### Option #2:
```

Comment:
I think I like this option better, but it still seems a bit weird.  Maybe it's good enough for now.  But it will definitely get out dated when SwiftUI comes along.